stubs/array_find.phpstub

```diff
@@ -0,0 +1,12 @@
+<?php
+
+/**
+ * @param array<TKey, TValue>                      $array
+ * @param callable(TValue $value, TKey $key): bool $callback
+ *
+ * @return TValue|null
+ *
+ * @template TKey of array-key
+ * @template TValue
+ */
+function array_find(array $array, callable $callback): mixed {}
```

Comment:
Why is a stub needed for this function but not the others?

Reply:
Psalm does not know about the new functions yet. We need this stub, so Psalm can infer the correct return type for `array_fInd()`. The other two new functions I've used always return a boolean, so the native return types might be just good enough.